pages/glossary/adjacent.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Adjacent
+key: adjacent
+unambiguous: true
+objective: true
+input_aspects:
+  - CSS styling
+  - DOM tree
+---
+
+[Bounding boxes][] A and B are _adjacent_ if at least one point from box A is at the distance of 1 point from one point in box B.
```

Comment:
Problem with this definition is that it also includes overlapping elements.

Reply:
Fixed